docroot/modules/custom/yukon_migrate/migrations/nodes/yukon_migrate_landing_page_level_2_translations.yml

```diff
@@ -0,0 +1,83 @@
+id: yukon_migrate_landing_page_level_2_translations
+label: Landing Page Level 2
+migration_tags:
+    - Drupal 7
+    - node
+    - Landing Page Level 2
+migration_group: legacy
+source:
+    plugin: d7_node_entity_translation
+    node_type: landing_page_level_2
+    constants:
+        default_text_format: full_html
+destination:
+    plugin: entity:node
+    default_bundle: landing_page_level_2
+    translations: true
+process:
+    uid: uid
+    nid:
+        plugin: migration_lookup
+        migration: yukon_migrate_landing_page_level_2
+        source: entity_id
+        no_stub: true
+    title: title
+    status:
+        plugin: skip_on_value
+        method: row
+        not_equals: true
+        value: 1
+        source: status
+    langcode: language
+    created: created
+    changed: changed
+    promote: promote
+    sticky: sticky
+    revision_log: log
+    field_page_description/value: field_page_description/0/value
+    field_page_description/format: constants/default_text_format
+    body/value: field_opener_description/0/value
+    body/format: constants/default_text_format
+    field_icon_dark:
+        plugin: migration_lookup
+        source: field_svg_upload/0/fid
+        migration: yukon_migrate_icon
+        no_stub: true
+    field_icon_light:
+        plugin: migration_lookup
+        source: field_light_svg_upload/0/fid
+        migration: yukon_migrate_icon
+        no_stub: true
+    pseudo_field_primary_content:
+        plugin: sub_process
+        source: field_primary_content
+        process:
+            paragraph:
+                plugin: migration_lookup
+                source: value
+                migration:
+                    - yukon_migrate_primary_content
+                no_stub: true
+
+    field_primary_content:
+        plugin: sub_process
+        source: '@pseudo_field_primary_content'
+        process:
+            target_id: 'paragraph/0'
+            target_revision_id: 'paragraph/1'
+    field_department_term:
+        plugin: migration_lookup
+        migration: yukon_migrate_department
+        source: field_department_term
+        no_stub: true
+    field_yukon_editorial_team:
+        plugin: migration_lookup
+        migration: yukon_migrate_teams
+        source: field_yukon_editorial_team
+        no_stub: true
+    field_social_sharing: field_social_sharing
+    content_translation_source: source
+    content_translation_outdated: translate
+migration_dependencies:
+    required:
+        - yukon_migrate_landing_page_level_2
```

Comment:
Can you please update the migration dependencies list and add here all required and optional migrations?
For example, your migration has the migration lookup in the yukon_migrate_teams migration, which means that this migration should be a dependency.
Could you please go through all your migrations and fix this? :)

Reply:
This has been resolved